### ISSUE TITLE
Unable to install programs which require Admin

### ISSUE THREAD

**Issue by filiphorvat** [microsoft/winget-cli#2740]
### Brief description of your issue

Winget is unable to install programs which require Admin.
Winget disables inheritance on the `C:\Users\username\AppData\Local\Temp\WinGet` folder so the administrator accounts can't access it.
Even after manually enabling inheritance, the next `winget install` disables it again and fails.

### Steps to reproduce

`winget install` any program that requires an Admin password

### Expected behavior

The program installs.

### Actual behavior

The program fails installing with the error `The network name cannot be found`, or similar.

### Environment

```shell
Windows Package Manager (Preview) v1.4.3132-preview
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19045.2311
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.19.3132.0
```

**Comment by denelon** [comment]
@filiphorvat Can you run an instance of this with `winget install <package> --verbose-logs` and share the log file? The location is displayed via `winget --info`.

**Comment by filiphorvat** [comment]
Here it is for `winget install wiztree --verbose-logs`:
[WinGet-2022-12-05-18-22-32.594.log](https://github.com/microsoft/winget-cli/files/10156465/WinGet-2022-12-05-18-22-32.594.log)
Looks like this after typing in the admin password:
![image](https://user-images.githubusercontent.com/28711765/205703003-dab04b8b-7d79-475d-9ab5-3421af9d683f.png)

Or like this when installing Steam:
![image](https://user-images.githubusercontent.com/28711765/205703900-2daafbfe-7fd3-4696-95e7-d01f1f449a59.png)
Here's the log for Steam:
[WinGet-2022-12-05-18-34-06.014.log](https://github.com/microsoft/winget-cli/files/10156527/WinGet-2022-12-05-18-34-06.014.log)
Even manually running the installer from the temp directory doesn't work if I don't re-enable inheritance.

**Comment by AlexDe74** [comment]
same as here https://github.com/microsoft/winget-cli/issues/2665

MS ignores it.

To install any programs with EXE Installers I do: 
1. Try first time winget update --id....
2. Go into %TEMP%\Winget\
3. Give to Administrator the permissions to access the applications's folder in %TEMP%\Winget
4. Try second time winget upgrade --id...

Really good user experience....

**Comment by ghborger** [comment]
The same behavior for me on Windows 11. Normal user without administrator rights tries to install a program with winget. Winget loads the installation files
to C:\Users\USERNAME\AppData\Local\Temp\WinGet, sets own permissions. Full access for the current account and system and removes inheritance. The local administrator has no rights to the directory. Then WinGet requests the administrator rights. After that winget aborts with an error that it cannot access the installation files. Which was predictable if the local administrator lacks the rights. Apparently the Winget developers always work with administrator rights, otherwise they would have noticed the error immediately.

**Comment by rand0muser123** [comment]
Can confirm we are able to replicate it on our tenant with onenote as it uses an exe
As a workaround we are downloading it from the ms website
It happens both on the store or through winget

**Comment by boozeman** [comment]
Hi,

How about the feature where you can set the download directory for winget packages in settings file? Some directory where all the users has rw-rights.

**Comment by nk111** [comment]
I can confirm that this is fixed in 1.5 preview of winget. I really wonder why this bug even made it to a release. Does nobody really work without admin rights?

**Comment by Masamune3210** [comment]
> I can confirm that this is fixed in 1.5 preview of winget. I really wonder why this bug even made it to a release. Does nobody really work without admin rights?

Principle of least privilege says you shouldn't

**Comment by UnidentifiedTag** [comment]
same error

![error](https://github.com/microsoft/winget-cli/assets/54625394/3926f7bc-3aeb-4264-a81d-8bcd0db1aa6b)
